src/main/kotlin/net/maizegenetics/phgv2/cli/CreateMafVcf.kt

```diff
@@ -288,43 +285,51 @@ class CreateMafVcf : CliktCommand(help = "Create g.vcf and h.vcf files from Anch
 
     /**
      * Function to see if the BED region is fully contained within a VariantContext
+     * Indels are left-justified
      * Bed:       |---|
      * Var: |--------------|
      */
     fun bedRegionContainedInVariant(region: Pair<Position,Position>, variant: VariantContext) : Boolean {
-        return variant.contig == region.first.contig && variant.start <= region.first.position && variant.end >= region.second.position
+        val end = if (variant.type == VariantContext.Type.SYMBOLIC) variant.end else variant.start
+        return variant.contig == region.first.contig && variant.start <= region.first.position && end >= region.second.position
     }
 
     /**
      * Function to see if the VariantContext is fully contained within a BED region
+     * Indels are left-justified
      * Bed: |--------------|
      * Var:       |---|
      */
     fun variantFullyContained(region: Pair<Position,Position>, variant: VariantContext) : Boolean {
-        return variant.contig == region.first.contig && variant.start >= region.first.position && variant.end <= region.second.position
+        val end = if (variant.type == VariantContext.Type.SYMBOLIC) variant.end else variant.start
+        return variant.contig == region.first.contig && variant.start >= region.first.position && end <= region.second.position
     }
 
     /**
-     * Function to see if the variant is partially contained at the start of the BED region
-     * Bed:      |--------------|
-     * Var:    |---|
+     * Function to see if the start of the variant is partially contained in the BED region
+     * Indels are left-justified
+     * Bed: |--------------|
+     * Var:              |---|
```

Comment:
Functions variantPartiallyContainedStart() and variantPartiallyContainedEnd() were correct, but the diagrams in the comments above each were reversed?  What you've changed makes more sense based on the code that is checking the variant values against the region values.

Reply:
Yes. The functions were correct in behavior, but the comments describing the behavior were backwards (as I understood them) so I fixed it.